docs/usage.md

```diff
@@ -79,6 +82,31 @@ Sample files that can be used as illustrative references can be found in this re
 | demographic_race_two_or_more_races | str | Value of either `true` or `false` |
 | hispanic_or_latino_ethnicity | str | Value of either `true` or `false` |
 
+## `quiz_questions.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| assessment_id | int | Assessment ID that can be joined against `assessments.csv` |
+| question_number | int | The relative order number of the question  |
+| question_id | UUID | A unique identifier for the question number |
+
+## `quiz_question_contents.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| id | UUID | Question ID that can be joined against `quiz_questions.csv` |
+| text | str | The question's text |
+| type | str | The type of question (can be ['multichoice', 'multianswer', 'numerical', 'essay']) |
+
+## `quiz_multichoice_answer.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| id | int | A unique question answer id |
+| question_id | UUID | Question ID that can be joined against `quiz_questions.csv` |
+| text | str | The answer text |
+| grade | float | The percentage of the question's total points received for this answer |
+
```

Comment:
Sorry, I keep noticing "one last things" :joy: . I think this table is missing the [feedback column](https://github.com/openstax/raise-enclave/blob/7c0e19fe7b5006d6c560bec1e7bd8fd3e8d9ddef/enclave_mgmt/compile_models.py#L123).